packages/calcite-components/src/components/list-item/utils.ts

```diff
@@ -4,6 +4,12 @@ const listSelector = "calcite-list";
 const listItemGroupSelector = "calcite-list-item-group";
 const listItemSelector = "calcite-list-item";
 
+export function getListItemChildLists(slotEl: HTMLSlotElement): HTMLCalciteListElement[] {
```

Comment:
Sidebar: we should generalize these and move into `dom`. These are some pretty nice utils.